crates/router/tests/connectors/stax.rs

```diff
@@ -403,14 +610,11 @@ async fn should_fail_for_refund_amount_higher_than_payment_amount() {
                 refund_amount: 150,
                 ..utils::PaymentRefundType::default().0
             }),
-            get_default_payment_info(),
+            get_default_payment_info(None, create_customer_and_get_token().await),
         )
         .await
         .unwrap();
-    assert_eq!(
-        response.response.unwrap_err().message,
-        "Refund amount (₹1.50) is greater than charge amount (₹1.00)",
-    );
+    assert_eq!(response.response.unwrap_err().message, "No error message",);
```

Comment:
This should be actual error, return the error from create_customer_and_get_token()

Reply:
Done